C/Recursive Programs/add.c

```diff
@@ -0,0 +1,31 @@
+#include<stdio.h>
+
+int y;
+
+/*
+    Function to add two numbers and
+    return the result
+*/
+int add(int m, int n)
+{
+    if(n == 0)
+        return m;
+
+    /*
+        Recursion: adding 1, n times and 
+        then at the end adding m to it
+    */
+    y = add(m, n-1) + 1;
+    return y;   // return the result
+}
+
+int main()
+{
+    printf("\n\n\t\tStudytonight - Best place to learn\n\n\n");
```

Comment:
Looks like the code is copied from studytonight website. No point in copying bro :)

Anyways files look good. Will check all the files and merge it

Reply:
yeah sorry ,used to practice from there and directly copied from my old directory without edits.Really sorry again:)